src/main/webapp/js/index/form.js

```diff
@@ -14,6 +14,7 @@ $("#randomize-form").submit((event) => {
         userID = localStorage.getItem("user");
     }
     let queryStr = $(this).serialize() + `&lat=${lat}&lng=${lng}&user=${userID}`;
+    console.log(queryStr);

```

Comment:
remove this if this is leftover from debugging

Reply:
done